src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -212,7 +212,7 @@
           long-summary: |-
             These addons are available:
                 http_application_routing - configure ingress with automatic public DNS name creation.
-                monitoring - turn on Log Analytics monitoring. Requires "--workspace-resource-id".
+                monitoring - turn on Log Analytics monitoring. Uses the Log Analytics Default Workspace if it exists, else creates one. Specify the "--workspace-resource-id" to use existing workspace.
```

Comment:
I'd suggest minor wording changes here:
`Use the default Log Analytics Workspace if it exists, otherwise create one. Specify "--workspace-resource-id" to use an existing workspace.`

Reply:
Fixed the wording. I kept 'uses' instead of 'use' since the CLI command is going to figure the default workspace automatically not the user . Please let me know if the updated wording not clear we can discuss.